sympy/crypto/crypto.py

```diff
@@ -3324,3 +3325,139 @@ def decipher_bg(message, key):
         orig_msg += (m ^ b)
 
     return orig_msg
+
+################# Columnar Transposition Cipher #################
+
+def encipher_columnar(message, key):
+    """
+    Performs Regular Columnar Encryption on plaintext and returns ciphertext
+
+    Examples
+    ========
+
+    >> from sympy.crypto.crypto import encipher_columnar
+    >> message = "HELLO WORLD"
+    >> encipher_columnar(message,"SYM")
+    "LWLEHLODEORY"
+
+    Parameters
+    ==========
+
+    message : string, the message to be encrypted.
+    key : string, the key to be used for encryption.
```

Comment:
Does it work properly if the key contains duplicate alphabets?
I believe that it should be handled as exception

Reply:
Yes, it still encrypts the message properly. I have considered that case in the code from line 3374 to line 3385